rum_react/README.md

```diff
@@ -2,65 +2,135 @@
 
 ## Overview
 
-With the Datadog [React integration][1], resolve performance issues quickly in React components by:
+With the Datadog RUM React integration, resolve performance issues quickly in React components by:
 
 - Debugging the root cause of performance bottlenecks, such as a slow server response time, render-blocking resource, or an error inside a component
-- Automatically correlating React performance data with user journeys, AJAX calls to the server side, and logs
-- Alerting your engineering teams when crucial performance metrics for React (such as Core Web Vitals) fall below a threshold that results in a poor user experience
-
+- Automatically correlating web performance data with user journeys, HTTP calls, and logs
+- Alerting your engineering teams when crucial web performance metrics (such as Core Web Vitals) fall below a threshold that results in a poor user experience
 
 Monitor your React applications from end-to-end by:
 
 - Tracking and visualizing user journeys across your entire stack
-- Debugging the root cause of slow load times, which may be an issue with your React code, network performance, or underlying infrastructure 
+- Debugging the root cause of slow load times, which may be an issue with your React code, network performance, or underlying infrastructure
 - Analyzing and contextualizing every user session with attributes such as user ID, email, name, and more
 - Unifying full-stack monitoring in one platform for frontend and backend development teams
 
 ## Setup
 
-### Collect RUM events 
+Make sure to [setup Datadog RUM][1] in your application. When creating a new RUM application within the Datadog App, make sure to select React. You can also edit an existing RUM application and change its type to React. The Datadog App then shows you the instructions to setup the Browser SDK [RUM-React plugin][2].
 
-To start collecting Real User Monitoring events from your React application, see [React Monitoring][2].
+## Error Tracking
 
-### Collect traces 
+To track React component rendering errors, use one of the following:
 
-Your React application automatically sends traces to Datadog.
+- An `ErrorBoundary` component (see [React documentation][3]) that catches errors and reports them to Datadog.
+- A function that you can use to report errors from your own `ErrorBoundary` component.
 
-### Collect logs 
+#### `ErrorBoundary` usage
 
-To start forwarding your React application's logs to Datadog, see [React Log Collection][3].
+```javascript
+import { ErrorBoundary } from '@datadog/browser-rum-react'
 
-## Data Collected
+function App() {
+  return (
+    <ErrorBoundary fallback={ErrorFallback}>
+      <MyComponent />
+    </ErrorBoundary>
+  )
+}
 
-### Metrics
+function ErrorFallback({ resetError, error }: { resetError: () => void; error: unknown }) {
```

Comment:
Maybe is better to remove the types if is not adding any information.

Reply:
Thanks for the suggestion, I removed the types